docs/eigenda/integrations-guides/rollup-guides/op-stack/README.md

```diff
@@ -64,23 +65,19 @@ The following env config values should be set accordingly to ensure proper commu
 
 When you are ready to onboard your rollup to mainnet you can fill out the following form to get your keypair whitelisted: [https://forms.gle/niMzQqj1JEzqHEny9](https://forms.gle/niMzQqj1JEzqHEny9).
 
-## Security Guarrantees
+## Security Guarantees
 
 This setup provides Stage 0 security guarrantees without adding an unnecessary trust assumption on the EigenDA disperser. The EigenDA Proxy [docs page](../../dispersal/clients/eigenda-proxy.md) and [repo readme](https://github.com/Layr-Labs/eigenda-proxy/blob/main/README.md) explain how this is achieved.
 
 ### OP Stack DA Challenge Contract
 
-One new component of OP Alt-DA interface is the [DA challenge contract](https://specs.optimism.io/experimental/alt-da.html#data-availability-challenge-contract), which allows L2 assetholders to delay a data withholding attack executed by the sequencer or DA network. This integration does not implement a DA challenge contract, because it would not provide any additional security to OP Stack deployment without fault proofs enabled. This is because the OP Sequencer is a trusted entity that unilaterally controls the liveness and safety of the L2 chain.
+One new component of OP Alt-DA interface is the [DA challenge contract](https://specs.optimism.io/experimental/alt-da.html#data-availability-challenge-contract), which allows L2 assetholders to delay a data withholding attack executed by the sequencer or DA network.
 
-The chain's liveness assumption on the sequencer is unchanged, because if the sequencer posts an invalid commitment then any verifier nodes syncing from EigenDA will halt, as they'd be incapable of inspecting the batch. This is equivalent to the same DoS vector where the sequencer arbitrarily stops producing batches.
-
-The safety property is unchanged, because although the DA challenge contract could theoretically prevent a malicious sequencer from peforming a data withholding attack, in a stage 0 deployment such a sequencer would already have unilateral control of the bridge, and so would be able to steal all the funds in broad daylight, even with all the data available to prove that they violated the chain's safety.
-
-The EigenDA protocol integrations team has roadmap plans to implement an DA challenge contract with along fault proof support in order to provide full safety guarrantees to OP Stack x EigenDA deployments.
+The EigenDA team has roadmap plans to implement an DA challenge contract with along fault proof support in order to provide full safety guarantees to OP Stack x EigenDA deployments.
 
 ## Roadmap
 
-The EigenDA Rollup Integrations team is working to support OP Stack fault proofs by the end of summer 2024, and will post updates to [@eigen_da](https://x.com/eigen_da?lang=en).
+The EigenDA Rollup Integrations team is working to support OP Stack fault proofs and will post updates to [@eigen_da](https://x.com/eigen_da?lang=en).
```

Comment:
This will be for major announcement? Shall we post normal updates to rollup channel(s) that we have?

Reply:
absolutely